metadata/client.go

```diff
@@ -523,8 +539,8 @@ func (def TrainingSetDef) ResourceType() ResourceType {
 	return TRAINING_SET_VARIANT
 }
 
-func (client *Client) CreateTrainingSetVariant(ctx context.Context, def TrainingSetDef) error {
-	serialized := &pb.TrainingSetVariant{
+func (def TrainingSetDef) Serialize() *pb.TrainingSetVariant {
```

Comment:
Are protos serialized objects or just objects?

Reply:
they're the serialization of objects